content/id/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -0,0 +1,109 @@
+---
+title: Menggunakan Service untuk Ekpos Aplikasimu
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="id">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+	<main class="content">
+
+		<div class="row">
+			<div class="col-md-8">
+    		<h3>Tujuan</h3>
+				<ul>
+					<li>Belajar tentang Service di Kubernetes</li>
+					<li>Memahami bagaimana label dan objek LabelSelector berhubungan dengan sebuah Service</li>
+					<li>Ekpos sebuah aplikasi keluar klaster Kubernetes menggunakan sebuah Service</li>
+				</ul>
+			</div>
+
+			<div class="col-md-8">
+			<h3>Ikhtisar Service Kubernetes</h3>
+
+			<p><a href="/id/docs/concepts/workloads/pods/">Pod-Pod</a> Kubernetes itu fana. Pod pada kenyataannya punya <a href="/id/docs/concepts/workloads/pods/pod-lifecycle/">siklus hidup</a>. Ketika sebuah Node pekerja mati, Pod yang berjalan dalam Node itu juga hilang. <a href="/id/docs/concepts/workloads/controllers/replicaset/">ReplicaSet</a> mungkin kemudian secara dinamis mendorong klaster kembali ke keadaan yang diinginkan melalui penciptaan Pod baru untuk memasikan aplikasimu berjalan. Contoh lainnya, katakanlah ada <i>backend</i> pengolahan citra dengan 3 replika. Replika tersebut dapat ditukar; sistem <i>front-end</i> seharusnya tidak peduli tentang replika <i>backend</i> atau bahkan jika Pod hilang dan dibuat ulang. Kathe front-end system should not care about backend replicas or even if a Pod is lost and recreated. walaupun demikian, tiap Pod dalam sebuah klaster Kubernetes memiliki alamat IP unik, meskipun Pod-Pod itu berada dalam satu Node yang sama, sehingga seharusnya ada jalan untuk mencocokan perubahan antara Pod-Pod itu sehingga aplikasimu tetap berfungsi.</p>
```

Comment:
typo? and forget to delete :)
```suggestion
			<p><a href="/id/docs/concepts/workloads/pods/">Pod-Pod</a> Kubernetes itu fana. Pod pada kenyataannya punya <a href="/id/docs/concepts/workloads/pods/pod-lifecycle/">siklus hidup</a>. Ketika sebuah Node pekerja mati, Pod yang berjalan dalam Node itu juga hilang. <a href="/id/docs/concepts/workloads/controllers/replicaset/">ReplicaSet</a> mungkin kemudian secara dinamis mendorong klaster kembali ke keadaan yang diinginkan melalui penciptaan Pod baru untuk memastikan aplikasimu berjalan. Contoh lainnya, katakanlah ada <i>backend</i> pengolahan citra dengan 3 replika. Replika tersebut dapat ditukar; sistem <i>front-end</i> seharusnya tidak peduli tentang replika <i>backend</i> atau bahkan jika Pod hilang dan dibuat ulang.  Walaupun demikian, tiap Pod dalam sebuah klaster Kubernetes memiliki alamat IP unik, meskipun Pod-Pod itu berada dalam satu Node yang sama, sehingga seharusnya ada jalan untuk mencocokan perubahan antara Pod-Pod itu sehingga aplikasimu tetap berfungsi.</p>
```